tasks/gulp/__tests__/after-build-package.test.js

```diff
@@ -1,21 +1,30 @@
-const fs = require('fs')
+const { readFile } = require('fs/promises')
 const path = require('path')
-const util = require('util')
-
 const recursive = require('recursive-readdir')
```

Comment:
Didn't do this, but could probably replace this package with `getFilesByDirectory()`